guides/common/modules/con_content-view-environment-ordering-and-priority.adoc

```diff
@@ -0,0 +1,62 @@
+[id="content_view_environment_ordering_and_priority_{context}"]
+= Content view environment ordering and priority
+
+When you assign multiple content view environments to a host or activation key, the order you specify determines the priority of the content.
+An activation key with multiple content view environments assigns them to hosts during registration in the same order as they are stored on the activation key.
+
+.Importance of content view environment order
+The order of assigned content view environments is critical in scenarios involving repository conflicts.
+A conflict occurs when two or more content view environments contain a repository with the same repository label.
+For example:
+A host assigned to the content view environments `Library/cv1, dev/cv1` uses the following configuration:
+[options="nowrap" subs="+quotes"]
+----
+hammer host update --name rhel9.example.com --organization-id 1 \
+--content-view-environments “Library/cv1, dev/cv1”
+----
+Both content view environments include the `satellite-client-6-for-rhel-9-x86_64-rpms` repository.
+When you inspect the host's available repositories using the `subscription-manager repos` command, the output reflects the `Library/cv1` content view environment:
+[options="nowrap" subs="+quotes"]
+----
+-# subscription-manager repos
++----------------------------------------------------------+
+    Available Repositories in /etc/yum.repos.d/redhat.repo
++----------------------------------------------------------+
+Repo ID:   satellite-client-6-for-rhel-9-x86_64-rpms
+Repo Name: Red Hat Satellite Client 6 for RHEL 9 x86_64 (RPMs)
+Repo URL:  https://centos9-katello-devel-stable.example.com/pulp/content/Default_Organization/Library/cv1/content/dist/layered/rhel9/x86_64/sat-client/6/os
+Enabled:   0
```

Comment:
```suggestion
```

Please drop this stdout. If you really require it, then add it to a separate code block and explain what users see.

Reply:
This is being used as an example of what you see in the "Repo URL" (the fact that the URL contains `Library/cv1` which is the content view environment it comes from.)

Reply:
unresolved: We should not mix commands and stdout. I suggest that you split this:

```suggestion
----
+
This results in the following output:
+
[source, none, options="nowrap" subs="+quotes"]
----
+----------------------------------------------------------+
    Available Repositories in /etc/yum.repos.d/redhat.repo
+----------------------------------------------------------+
Repo ID:   satellite-client-6-for-rhel-9-x86_64-rpms
Repo Name: Red Hat Satellite Client 6 for RHEL 9 x86_64 (RPMs)
Repo URL:  https://centos9-katello-devel-stable.example.com/pulp/content/Default_Organization/Library/cv1/content/dist/layered/rhel9/x86_64/sat-client/6/os
Enabled:   0
```

The "source none" is important to prevent syntax highlighting etc.